shared/engine/index-impl.tsx

```diff
@@ -35,8 +36,31 @@ class Engine {
   _hasConnected: boolean = isMobile // mobile is always connected
   // App tells us when the sagas are done loading so we can start emitting events
   _sagasAreReady: boolean = false
+
+  static _realDispatch: Dispatch
```

Comment:
Why use `static` for these?

Reply:
its static in engine so other things can just use it and not have an instance (like session etc). Before this PR dispatch was static, now these helpers of dispatch are also

Reply:
Ah k, afaict they're not used outside this class (session uses `deprecatedGetDispatch`). Slightly confusing.